coach/src/concept-coach/index.cjsx

```diff
@@ -123,13 +123,16 @@ class ConceptCoachAPI extends EventEmitter2
       componentModel.channel.emit('close.clicked')
 
     @close = props.close
+
+    # Needs to be added on initialize since opening from a coach path calls
+    # initialize, and not open.  On will handle multiple logging in and out.
+    # 
+    # Wait until our logout request has been received and the close
+    User.channel.on('logout.received', @close)
+
     @component = coachWrapped.render(mountNode, props)
```

Comment:
I seem to remember that at one time we had problems with webview calling initialize being multiple times, which is why we added the `remove()` call on line 112 above.  

Should we add a `User.channel.off('logout.received', @close) to the `remove` method so they're balanced?